pilot/pkg/xds/testdata/benchmarks/gateways.yaml

```diff
@@ -67,6 +67,78 @@ spec:
     - random-c-{{$i}}.host.example
     tls:
       mode: ISTIO_MUTUAL
+  - port:
+      number: 443
+      name: https-simple-1
+      protocol: HTTPS
+    hosts:
+    - random-a-{{$i}}.host.example
+    - random-b-{{$i}}.host.example
+    - random-c-{{$i}}.host.example
+    tls:
+      mode: SIMPLE
+      serverCertificate: /etc/certs/servercert.pem
+      privateKey: /etc/certs/privatekey.pem
+  - port:
+      number: 443
+      name: https-simple-2
+      protocol: HTTPS
+    hosts:
+    - random-a-{{$i}}.host.example
+    - random-b-{{$i}}.host.example
+    - random-c-{{$i}}.host.example
+    tls:
+      mode: SIMPLE
+      serverCertificate: /etc/certs/servercert.pem
+      privateKey: /etc/certs/privatekey.pem
+  - port:
+      number: 443
+      name: https-simple-3
+      protocol: HTTPS
+    hosts:
+    - random-a-{{$i}}.host.example
+    - random-b-{{$i}}.host.example
+    - random-c-{{$i}}.host.example
+    tls:
+      mode: SIMPLE
+      serverCertificate: /etc/certs/servercert.pem
+      privateKey: /etc/certs/privatekey.pem
+  - port:
+      number: 443
+      name: https-simple-4
+      protocol: HTTPS
+    hosts:
+    - random-a-{{$i}}.host.example
+    - random-b-{{$i}}.host.example
+    - random-c-{{$i}}.host.example
+    tls:
+      mode: SIMPLE
+      serverCertificate: /etc/certs/servercert.pem
+      privateKey: /etc/certs/privatekey.pem
+  - port:
+      number: 443
+      name: https-simple-5
+      protocol: HTTPS
+    hosts:
+    - random-a-{{$i}}.host.example
+    - random-b-{{$i}}.host.example
+    - random-c-{{$i}}.host.example
+    tls:
+      mode: SIMPLE
+      serverCertificate: /etc/certs/servercert.pem
+      privateKey: /etc/certs/privatekey.pem
+  - port:
+      number: 443
+      name: https-simple-6
+      protocol: HTTPS
+    hosts:
+    - random-a-{{$i}}.host.example
```

Comment:
isn't this config invalid? we have overlapping host names?

Reply:
hmm. you are right. Let me revert this gateways.yaml change.